mkdocs.yml

```diff
@@ -108,7 +108,9 @@ nav:
       - ForPairs: api-reference/state/forpairs.md
       - ForValues: api-reference/state/forvalues.md
       - Observer: api-reference/state/observer.md
+      - peek: api-reference/state/peek.md
```

Comment:
Not sure if this is intentional, but `peek` is spelt lowercase while `use` is spelt uppercase.

Reply:
That's intentional - the `Use` specified in the docs here is a type definition, which by convention start with a capital.